core/client/src/main/java/alluxio/hadoop/HdfsFileInputStream.java

```diff
@@ -160,7 +160,7 @@ public int read() throws IOException {
       throw new IOException("Cannot read from a closed stream.");
     }
     if (mAlluxioFileInputStream != null) {
-      int ret = 0;
+      int ret;
```

Comment:
I think we could move the declaration down to the line where `ret` is initialized